### PR TITLE
Reduced the -t so it will not timeout in ZTS

### PR DESCRIPTION
**Pull request by jsalinasintel**

### CODE REVIEW

File: tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh [hunk 1]
```diff
@@ -36,6 +36,6 @@
 #	is set to 5min, but actual runtime might be longer.
 #
 
-log_must $RAIDZ_TEST -S -t 300
+log_must $RAIDZ_TEST -S -t 40
```
Comment (behlendorf) [inline tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh]: In exactly what environment does this need to be reduced.  Recent automated test runs from master show that this test does take roughly 300 seconds to run.  

```
Test: /usr/share/zfs/zfs-tests/tests/functional/raidz/raidz_002_pos (run as root) [05:04] [PASS]
```

**Comment by ironMann** [comment]
Issue with wall time resolved in PR #5190. The test should only run 1-2s longer than specified.

File: tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh (hunk 1 above)
Comment (ironMann) [inline tests/zfs-tests/tests/functional/raidz/raidz_002_pos.ksh]: This test processes 'work-units' which are getting progressively bigger with time. So, when run on a fast CPU, with lot of cores, wall time discrepancy can get worse as time passes. See #5190 for fix.

**Comment by behlendorf** [comment]
@ironMann that makes perfect sense and explains when it was mainly observed on real hardware.  Thanks for the prompt fix.  Closing in favor of #5190.

**Pull request closed without merging**